ci/request-reviews/request-reviews.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# Requests reviews for a PR after verifying that the base branch is correct
+
+set -euo pipefail
+tmp=$(mktemp -d)
+trap 'rm -rf "$tmp"' exit
+SCRIPT_DIR=$(dirname "$0")
+
+log() {
+    echo "$@" >&2
+}
+
+if (( $# < 3 )); then
+    log "Usage: $0 GITHUB_REPO PR_NUMBER OWNERS_FILE"
+    exit 1
+fi
+baseRepo=$1
+prNumber=$2
+ownersFile=$3
+
+log "Fetching PR info"
+prInfo=$(gh api \
+  -H "Accept: application/vnd.github+json" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
```

Comment:
Worth turning this into a helper function (given the same API version header is used below) and/or document why we need to specify API version?

Reply:
This is how the API calls are shown in the docs (e.g. [here](https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#get-a-pull-request)). I guess this might be intentional, because if a new version is introduced, keeping the version for each call allows you to use the new version without having to update all of them at once.

I don't think it's worth abstracting this either way though, since there's [less than three](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)).